src/main/webapp/index.html

```diff
@@ -78,7 +79,7 @@ <h1>Census Viz
               autocomplete="off" required onkeyup="validateInput('location')"
               onfocus="storeValueAndEmpty('location')"
               onfocusout="replaceValueIfEmpty('location')"
-              value="each U.S. state">
+              value="Every U.S. state">
```

Comment:
is there a reason for this change? it looks like the dropdown list creator is still making the dropdown item "each U.S. state", and I feel like the dropdown item and the initial value should be the same.

Reply:
also, it makes more sense not capitalized since it is part of the overall query sentence.

Reply:
I actually don't remember making this change. Maybe I pulled from an old version of the master? Either way, fixed so it is reverted back.